formatter-pretty/package.json

```diff
@@ -26,5 +26,8 @@
   },
   "keywords": [
     "markdownlint-cli2-formatter"
-  ]
+  ],
+  "engines": {
+    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
```

Comment:
This is required for the Chalk 5 dependency, but it also then let a few disable statements get striped out

Reply:
Per https://github.com/DavidAnson/markdownlint-cli2/issues/72#issuecomment-1283347887, it looks like I'll be changing CLI2 to be `>=14.18.0`. Can we use the same thing here for consistency?